actioncable/test/client/authorization_test.rb

```diff
@@ -5,18 +5,20 @@ class ActionCable::Connection::AuthorizationTest < ActionCable::TestCase
   class Connection < ActionCable::Connection::Base
     attr_reader :websocket
 
-    def connect
```

Comment:
Is there a way to keep this type of code inside of `ActionCable::Connection::Base`? Otherwise, when, for example,  generating a new Rails application we'll have to create three `ApplicationCable` files: `Connection`, `Client`, and `Channel`...